### DISCUSSION TITLE
add the Search Function

### DISCUSSION THREAD

**Discussion by vakkie** [maximevaillancourt/digital-garden-jekyll-template#188] [Ideas]
Hi, sir, this template is very awesome. Is it possible to add the search function, like this? many thanks!
![image](https://github.com/maximevaillancourt/digital-garden-jekyll-template/assets/92929085/4baba0db-8436-41c6-80f8-b56e7857838d)

**Comment by jameswjesso** [comment]
I would also love to know how to do this!

**Comment by PlexSheep** [comment]
I too would find that useful.
